pages/api/stonks/create-off-chain-order.ts

```diff
@@ -0,0 +1,25 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { fetchCowApi } from 'modules/stonks/utils/fetchCowApi'
+
+export default async function createOffChainOrder(
+  req: NextApiRequest,
+  res: NextApiResponse,
+) {
+  try {
+    const uid = await fetchCowApi<string>({
+      chainId: req.query.chainId,
+      url: '/orders',
+      method: 'POST',
+      body: JSON.stringify(req.body),
+    })
+
+    res.status(201).json(uid)
+  } catch (error) {
+    const errorMessage =
+      error instanceof Error
+        ? error.message
+        : 'Error occured while creating order'
+    console.error(errorMessage, error)
+    res.status(500).send({ error: errorMessage })
```

Comment:
Looks like we could use json here instead of send to have same logic like in line 16
res.status(500).json({ error: FALLBACK_ERROR })